### PR TITLE
Pull request for feedback

### PR DESCRIPTION
**Pull request by kardokiyani**
Please review my social media site.

### CODE REVIEW

File: login.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        <div class="mb-3">
+          <label for="exampleInputPassword1" class="form-label">Password</label>
+          <input type="password" class="form-control" id="exampleInputPassword1">
+        </div>
+        <div class="mb-3 form-check">
+          <input type="checkbox" class="form-check-input" id="exampleCheck1">
+          <label class="form-check-label" for="exampleCheck1">Check me out</label>
+        </div>
+        <button type="submit" class="btn btn-primary">Submit</button>
+      </form>
```
Comment (jaaneh) [inline login.html]: Your form action attribute should be set to `/profile.html` like the assignment brief asks for.

You should perform some client-side validation in your form. Things like `required`, `minlength`, `pattern` attributes. See [MDN Docs](https://developer.mozilla.org/en-US/docs/Learn/Forms/Form_validation).

File: index.html [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <div class="formGroup">
+        <label for="comment">Create a new post:</label>
+        <textarea class="form-control" rows="5" id="comment"></textarea>
+      </div>
```
Comment (jaaneh) [inline index.html]: You should be using the `<form>` tag here.
You should also add a button for submitting the post.

Comment (kardokiyani) [reply index.html]: I have changed and fixed them now.

File: login.html (hunk 1 above)
Comment (kardokiyani) [reply login.html]: I have changed and fixed that now.

Comment (jaaneh) [reply login.html]: Your changes look good. You can resolve this conversation.

File: index.html (hunk 2 above)
Comment (jaaneh) [reply index.html]: Your changes look good. You can resolve this conversation.

**Comment by jaaneh** [comment]
Looks good to me. You've implemented the changes I suggested and you're free to merge this PR :-)

**Pull request merged**